src/libserialize/serialize.rs

```diff
@@ -706,6 +706,30 @@ impl<T: Decodable> Decodable for Vec<T> {
     }
 }
 
+impl Encodable for [u8; 20] {
```

Comment:
This is compiler-internal so you can just use const generics, i.e.:
```rust
impl<T: Encodable, const N: usize> Encodable for [T; N] {
```
I doubt the body needs to change.

Reply:
On second thought, `Decodable` won't work like this for now, so you can leave these changes alone.